samples/CMakeLists.txt

```diff
@@ -5,8 +5,8 @@
 #
 
 if (CONFIG_SECURE_BOOT)
-  zephyr_add_executable(b0 require_build)
-  if (${require_build})
-    add_subdirectory(bootloader)
-  endif ()
+	zephyr_add_executable(b0 require_build)
```

Comment:
?

Reply:
So the idea here is to check if `CONFIG_SECURE_BOOT` is enabled and if it creates `b0` as a separate target. I know the formatting is currently a bit weird as I'm working out some dependencies kinks in the build. This is done this way so that you could build bootloader as a sample or as a part of a multi-image build.

Maybe this is also not the correct place to put this code. It could be put in a directory above and do `samples/bootloader`  instead? That would probably be favored over adding a new `CMakeLists.txt` file?

Reply:
Sorry, the diff was different when I commented, then the only diff was indentation, and the indentation was wrong.

Reply:
Looks like the indentation is still wrong though

Reply:
>  know the formatting is currently a bit weird as I'm working out some dependencies kinks in the build.

Sorry, I see you have addressed this